### ISSUE TITLE
After #721 compiled size too big for OTA on 1M

### ISSUE THREAD

**Issue by zewelor** [Aircoookie/WLED#740]
After this: https://github.com/Aircoookie/WLED/pull/721/files#diff-237285c047dad808f73698c139144e1aR92 image gets too big. I think including bearssl shouln't be done by default, or maybe only on bigger boards ?

**Comment by Aircoookie** [comment]
Bearssl isn't included at all as far as I know, it should only matter if SSL libraries are actually used, which is not the case in WLED right now.

You are right though that the previous `esp01_1m` environment didn't match any of the release binaries (disabled some features to make space but still considerably too big for OTA), so I split it into `esp01_1m_full` and `esp01_1m_ota` environments.
